lib/main.html

```diff
@@ -57,6 +57,10 @@
     opacity: 1 !important;
     pointer-events: auto !important;
   }
+  .CodeMirror-ruler {
+    border-left-color: rgba(142, 142, 142, 0.5);
+    mix-blend-mode: difference;
```

Comment:
overwrite the default CodeMirror Ruler color.

The default ruler color is #ccc which is too bright in dark themes, so I made this tweak to ensure the ruler fits all kind of themes.